spk/tt-rss/src/dsm-control.sh

```diff
@@ -5,81 +5,23 @@ PACKAGE="tt-rss"
 DNAME="Tiny Tiny RSS"
 
 # Others
-INSTALL_DIR="/usr/local/${PACKAGE}"
-WEB_DIR="/var/services/web"
+INSTALL_DIR="/var/packages/${PACKAGE}/target"
 PATH="${INSTALL_DIR}/bin:${PATH}"
-BUILDNUMBER="$(/bin/get_key_value /etc.defaults/VERSION buildnumber)"
-USER="$([ "${BUILDNUMBER}" -ge "4418" ] && echo -n http || echo -n nobody)"
+USER="http"
 
-TTRSS="${WEB_DIR}/${PACKAGE}/update.php"
-HT_ACCESS_FILE="${WEB_DIR}/${PACKAGE}/.htaccess"
 PID_FILE="${INSTALL_DIR}/var/tt-rss.pid"
-HT_ACCESS_SECTION_DELIMITER="#Synology PHP"
-
```

Comment:
This section was actually useless since WebStation enforces it using the profile mechanisms.